package.json

```diff
@@ -6,12 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "less": "lessc -x src/evolve.less evolve/evolve.css",
-    "dev": "npx webpack --config webpack.evolve.dev.js",
-    "evolve": "npx webpack --config webpack.evolve.prod.js",
-    "build": "npx webpack --config webpack.evolve.prod.js && npx webpack --config webpack.wiki.prod.js",
-    "build-dev": "npx webpack --config webpack.evolve.dev.js && npx webpack --config webpack.wiki.dev.js",
-    "wiki-dev": "npx webpack --config webpack.wiki.dev.js",
-    "wiki": "npx webpack --config webpack.wiki.prod.js",
+    "build": "node buildEvolve.js",
```

Comment:
Is it possible to use the esbuild CLI instead to avoid these extra files?

Reply:
Yeah, you should be able to. I don't think there's necessarily a reason to remove the file though, it wouldn't affect the bundle size.